src/util/formatTrialTax.js

```diff
@@ -0,0 +1,11 @@
+const formatTrialTax = (price, tax, incTax) => {
+  const taxValue = price * tax;
+  const isGrossAndTaxEqual = incTax === Number(price + tax * price).toFixed(2);
```

Comment:
let's rename isGrossAndTaxEqual to something like `shouldRoundTaxValue`
+ 
`const truncatedTaxValue = (Math.trunc(taxValue * 100) / 100).toFixed(2)`;
return truncatedTaxValue;